### PR TITLE
DOCS-3211 Document new + Add a backup credential button

### PR DESCRIPTION
**Pull request by tonyriv3**
DOCS-3211 Document new + Add a backup credential button

### CODE REVIEW

File: content/SCALE/DataProtection/Cloud Sync Tasks/_index.md [hunk 1]
```diff
@@ -5,19 +5,19 @@ weight: 20
 
 {{< toc >}}
 
-TrueNAS can send, receive, or synchronize data with a cloud storage provider. Cloud sync tasks allow for single time transfers or recurring transfers on a schedule and are an effective method to back up data to a remote location.
+TrueNAS can send, receive, or synchronize data with a cloud storage provider. Cloud sync tasks allow for single-time transfers or recurring transfers on a schedule. They are an effective method to back up data to a remote location.
 
 {{< hint warning >}}
-Using the cloud means that data can go to a third party commercial vendor not directly affiliated with iXsystems. Investigate and fully understand that cloud vendor’s pricing policies and services before creating any cloud sync task. iXsystems is not responsible for any charges incurred from the use of third party vendors with the Cloud Sync feature.
+Using the cloud means data can go to a third-party commercial vendor not directly affiliated with iXsystems. You should fully understand vendor pricing policies and services before using them for cloud sync tasks. iXsystems is not responsible for any charges incurred from using third-party vendors with the Cloud Sync feature.
```
Comment (micjohnson777) [inline content/SCALE/DataProtection/Cloud Sync Tasks/_index.md:11]:
```suggestion to draw more attention to it, placed last line in separate paragraph
Using the cloud means data can go to a third-party commercial vendor not directly affiliated with iXsystems. You should fully understand vendor pricing policies and services before using them for cloud sync tasks.

 iXsystems is not responsible for any charges incurred from using third-party vendors with the cloud sync feature.
```

File: content/SCALE/DataProtection/Cloud Sync Tasks/_index.md [hunk 2]
```diff
@@ -5,19 +5,19 @@ weight: 20
...
 {{< /hint >}}
 
-TrueNAS supports major providers like Amazon S3, Google Cloud, and Microsoft Azure, along with a variety of other vendors. To see the full list of supported vendors, go to **Credentials > Backup Credentials > Cloud Credentials** click **Add** and open the **Provider** dropdown.
+TrueNAS supports major providers like Amazon S3, Google Cloud, and Microsoft Azure. It also supports many other vendors. To see the full list of supported vendors, go to **Credentials > Backup Credentials > Cloud Credentials** click **Add** and open the **Provider** dropdown.
```
Comment (micjohnson777) [inline content/SCALE/DataProtection/Cloud Sync Tasks/_index.md:14]:
```suggestion
TrueNAS supports major providers like Amazon S3, Google Cloud, and Microsoft Azure. It also supports many other vendors. To see the full list of supported vendors, go to **Credentials > Backup Credentials > Cloud Credentials** click **Add** and open the **Provider** drop-down list.
```

File: content/SCALE/DataProtection/Cloud Sync Tasks/_index.md [hunk 3]
```diff
@@ -5,19 +5,19 @@ weight: 20
...
 
 ## Requirements
 
-* All system [Storage](/scale/storage/) must be configured and ready to receive or send data.
-* A cloud storage provider account and a cloud storage location, like an Amazon S3 bucket, must be available.
-* A cloud storage account credentials must be saved to **System > Cloud Credentials** before creating the sync task. See *Cloud Credentials* for specific instructions.
+* You must have all system [Storage](/scale/storage/) configured and ready to receive or send data.
```
Comment (micjohnson777) [inline content/SCALE/DataProtection/Cloud Sync Tasks/_index.md:18]:
```suggestion
* You must have all system [storage](/scale/storage/) configured and ready to receive or send data.
```

File: content/SCALE/DataProtection/Cloud Sync Tasks/_index.md [hunk 4]
```diff
@@ -29,32 +29,32 @@ Go to **Data Protection > Cloud Sync Tasks** and click **Add**.
...
+**Copy** duplicates each source file into the destination, overwriting any destination files using the same name as the source. Copying is the least potentially destructive option.
 {{< /tab >}}
 {{< tab "Move" >}}
-**Move** transfers the files from the source to the destination and _deletes_ the original source files. Files with the same names on the destination are overwritten.
+**Move** transfers the files from the source to the destination and deletes the source files. **Move** also overwrites files with the same names on the destination.
 {{< /tab >}}
 {{< /tabs >}}
 
-Next, Use the **Control** options. Define when the task runs using **Schedule**. When a specific schedule is required, select **Custom** and use the **Advanced Scheduler**.
+Next, Use the **Control** options. Define when the task runs using **Schedule**. If you need a specific schedule, select **Custom** and use the **Advanced Scheduler**.
```
Comment (micjohnson777) [inline content/SCALE/DataProtection/Cloud Sync Tasks/_index.md:51]:
```suggestion
Next, use the **Control** options. Define when the task runs using **Schedule**. If you need a specific schedule, select **Custom** and use the **Advanced Scheduler**.
```

File: content/SCALE/DataProtection/Cloud Sync Tasks/_index.md [hunk 5]
```diff
@@ -29,32 +29,32 @@ Go to **Data Protection > Cloud Sync Tasks** and click **Add**.
...
 
 {{< expand "Advanced Scheduler" "v" >}}
 {{< include file="static/includes/SCALE/SCALEAdvancedScheduler.md.part" markdown="true" >}}
 {{< /expand >}}
 
-Remove the checkmark from the **Enable** checkbox to make the configuration available without allowing the specified schedule to run the task. To manually activate a saved task, go to **Data Protection > Cloud Sync Tasks**, click <i class="fa fa-play" aria-hidden="true"></i> for the cloud sync task you want to run.  You are prompted to **CONTINUE** or **CANCEL** the **Run Now** operation.
+Uncheck the **Enable** checkbox to make the configuration available without allowing the specified schedule to run the task. To manually activate a saved task, go to **Data Protection > Cloud Sync Tasks**, click <i class="fa fa-play" aria-hidden="true"></i> for the cloud sync task you want to run.  You are prompted to **CONTINUE** or **CANCEL** the **Run Now** operation.
```
Comment (micjohnson777) [inline content/SCALE/DataProtection/Cloud Sync Tasks/_index.md:57]:
```suggestion
Clear the **Enable** checkbox to make the configuration available without allowing the specified schedule to run the task. To manually activate a saved task, go to **Data Protection > Cloud Sync Tasks**, click <i class="fa fa-play" aria-hidden="true"></i> for the cloud sync task you want to run.  You are prompted to **CONTINUE** or **CANCEL** the **Run Now** operation.
```

File: content/SCALE/DataProtection/Cloud Sync Tasks/_index.md [hunk 6]
```diff
@@ -63,7 +63,7 @@ The remaining options allow tuning the task to your specific requirements.
 
 ### Scripting and Environment Variables
 
-Advanced users can write scripts that run immediately *before* or *after* the Cloud Sync task. The **Post-script** field is only run when the cloud sync task successfully completes. You can pass a variety of task environment variables into the **Pre-** and **Post-** script fields:
+Advanced users can write scripts that run immediately *before* or *after* the Cloud Sync task. The **Post-script** field only runs when the cloud sync task succeeds. You can pass a variety of task environment variables into the **Pre-** and **Post-** script fields:
```
Comment (micjohnson777) [inline content/SCALE/DataProtection/Cloud Sync Tasks/_index.md:66]:
```suggestion
Advanced users can write scripts that run immediately before or after the cloud sync task. The **Post-script** field only runs when the cloud sync task succeeds. You can pass a variety of task environment variables into the **Pre-** and **Post-** script fields:
```

File: content/SCALE/DataProtection/Cloud Sync Tasks/_index.md [hunk 7]
```diff
@@ -88,7 +88,7 @@ Local storage settings:
 
 ### Testing Settings
 
-To test the settings before saving, click **Dry Run**. TrueNAS connects to the cloud storage provider and simulates a file transfer. No data is sent or received.
+To test the settings before saving, click **Dry Run**. TrueNAS connects to the cloud storage provider and simulates a file transfer but does not send or receive data.
 A dialog box displays with the test status and allows you to download the task logs. You can also run this by clicking <i class="material-icons" aria-hidden="true" title="Dry Run">loop</i> after the task is created.
```
Comment (micjohnson777) [inline content/SCALE/DataProtection/Cloud Sync Tasks/_index.md:92]:
```suggestion
A dialog box displays with the test status and allows you to download the task logs. You can also run this by clicking <i class="material-icons" aria-hidden="true" title="Dry Run">loop</i> after creating the task.
```

File: content/SCALE/DataProtection/Cloud Sync Tasks/_index.md [hunk 8]
```diff
@@ -97,16 +97,16 @@ A dialog box displays with the test status and allows you to download the task l
 
 Saved tasks activate according to their schedule or by clicking <i class="material-icons" aria-hidden="true" title="Run Now">play_arrow</i>. An in-progress cloud sync must finish before another can begin. Stopping an in-progress task cancels the file transfer and requires starting the file transfer over.
 
-To view logs about a running or the most recent run of a task, click the task *State*.
+To view logs about a running task, or its most recent run, click *State*.
```
Comment (micjohnson777) [inline content/SCALE/DataProtection/Cloud Sync Tasks/_index.md:100]:
```suggestion
To view logs about a running task, or its most recent run, click **State**.
```

**Review by micjohnson777 — COMMENTED** [review]
Just a few things found with tense/voice and style. Otherwise LGTM!

**Comment by tonyriv3** [comment]
@micjohnson777 
I approved all the commits you suggested. This one is ready to merge!

**Review by micjohnson777 — APPROVED** [review]

**Pull request merged**